### ISSUE TITLE
Lovelace Card not getting updated with recent state value

### ISSUE THREAD

**Issue by Xerrathbg** [denysdovhan/purifier-card#75]
**Before submitting a bug**

[ x ] I updated to the latest version available
[ x ] I cleared the cache of my browser

**Describe the bug**

I managed to configure the card quite nicely with Xiaomi Air Purifier 3c and everytihng is working properly (will add the code below), even the AQI is working, but the issue is that the states are not getting updated regularly (AQI, RPMs, Filter %, etc.). If I turn it on/off, interact with the card in any way or reload the browser, all states will get updated immediately, but if I leave it like that, nothing gets updated.
Also if I call a service, to put on the child lock like:
```
shortcuts:
  - name: Child Lock
    icon: mdi:lock
    service: switch.toggle
    service_data:
      entity_id: switch.mi_air_purifier_3c_2_child_lock
```
It works perfectly, but I get the loading circle and the card stops responding. It fixes itself on browser reload.

![image](https://user-images.githubusercontent.com/91604432/142886673-8dd37a6c-e1ad-4eab-b331-3a12f304ee05.png)

**Steps to reproduce**

1. Install repo
2. Add lovelace card with the following code:
```
type: custom:purifier-card
entity: fan.mi_air_purifier_3c_1
stats:
  - entity_id: sensor.mi_air_purifier_3c_1_filter_life_remaining
    unit: '%'
    subtitle: Filter Remaining
  - entity_id: sensor.mi_air_purifier_3c_1_motor_speed
    unit: rpm
    subtitle: Motor Speed
shortcuts:
  - name: Auto
    icon: mdi:brightness-auto
    preset_mode: Auto
  - name: Silent
    icon: mdi:weather-night
    preset_mode: Silent
  - name: 25%
    icon: mdi:circle-slice-2
    preset_mode: Favorite
    service: number.set_value
    service_data:
      entity_id: number.mi_air_purifier_3c_1_favorite_motor_speed
      value: '750'
  - name: 50%
    icon: mdi:circle-slice-4
    preset_mode: Favorite
    service: number.set_value
    service_data:
      entity_id: number.mi_air_purifier_3c_1_favorite_motor_speed
      value: '1250'
  - name: 75%
    icon: mdi:circle-slice-6
    preset_mode: Favorite
    service: number.set_value
    service_data:
      entity_id: number.mi_air_purifier_3c_1_favorite_motor_speed
      value: '1700'
  - name: 100%
    icon: mdi:circle-slice-8
    preset_mode: Favorite
    service: number.set_value
    service_data:
      entity_id: number.mi_air_purifier_3c_1_favorite_motor_speed
      value: '2200'
  - name: Child Lock
    icon: mdi:lock
    service: switch.toggle
    service_data:
      entity_id: switch.mi_air_purifier_3c_2_child_lock
show_name: true
show_state: true
show_toolbar: true
compact_view: true
aqi:
  entity_id: sensor.mi_air_purifier_3c_1_pm2_5
  unit: µg/m³

```
**Expected behavior**
The card to get updated on a regular basis with most recent state values.

**Versions:**

- Release: v1.0.1
- Browser: Chrome & Fully Kiosk
- HomeAssistant version: core-2021.11.5

**Comment by mortenmoulder** [comment]
I can replicate the same behavior in Chrome on Windows. The buttons work perfectly, but the spinner never hides itself

**Comment by austwhite** [comment]
I think this could be related to the issue previously opened with values not updating

**Comment by austwhite** [comment]
Could this be related to the deprecated set fan speed service?
```
Logger: homeassistant.components.fan
Source: components/fan/__init__.py:246
Integration: Fan (documentation, issues)
First occurred: November 30, 2021, 7:59:25 PM (42 occurrences)
Last logged: 9:29:04 AM

The fan.set_speed service is deprecated, use fan.set_percentage or fan.set_preset_mode instead
```

Unsure if this would cause the issue.  The deprecated service actually does not work at all in 2021.11.x versions

**Comment by denysdovhan** [comment]
@austwhite the issue you are describing is fixed in the latest release 2.0.0. The card got some major changes, so hopefully the issue will be resolved. Please, reopen if you still have this issue after upgrading to the latest version.

**Comment by austwhite** [comment]
@denysdovhan    Absolutely perfect again now on my system.
Appreciate your help with this.     I'll send you a donation later in the week after I return from my business trip :)
